dist/stat_data.py

```diff
@@ -723,7 +723,9 @@ def __init__(self, name, desc, flags=''):
     RecStat('rec_multiblock_leaf', 'leaf page multi-block writes'),
     RecStat('rec_multiblock_max', 'maximum blocks required for a page', 'max_aggregate,no_scale'),
     RecStat('rec_overflow_key_internal', 'internal-page overflow keys'),
+    RecStat('rec_overflow_key_internal_hs', 'internal-page overflow keys in history store'),
```

Comment:
These are data source stats - I thought only connection wide ones would be reflected in FTDC?